lib/govuk_app_config/configure.rb

```diff
@@ -42,3 +42,14 @@
     GovukStatsd.increment("error_reports_failed")
   }
 end
+
+Rails.application.config.xray = {
```

Comment:
Might be nice to extract this into an `xray.rb` file and then include it.

Reply:
Done